contrib/test/ci/vars.yml

```diff
@@ -88,6 +88,7 @@ kata_int_test_env:
     PRIVILEGED_WITHOUT_HOST_DEVICES: true
     RUNTIME_CONFIG_PATH: "/opt/kata/share/defaults/kata-containers/configuration.toml"
     PATH: "/opt/kata/bin:{{ ansible_env.PATH }}"
+    RUNTIME_PULL_IMAGE: true
```

Comment:
Actually, I think we should keep it "false" here (or not set it at all as it is the default)
Our testing is done on kata alone - not peer-pods nor CoCo. We don't want to enable that feature systematically.